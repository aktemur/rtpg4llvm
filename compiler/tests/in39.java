/*compunit*/
double rows;
double *rows;
double *rows[3];
double rows = 5.5;
double rows[3];
double rows[3] = -5.5;
