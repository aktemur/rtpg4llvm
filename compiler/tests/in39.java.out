mkProgram(6, new GlobalDecl(new DoubleType(), "rows"),
   new GlobalDecl(new PtrType(new DoubleType()), "rows"),
   new GlobalDecl(new ArrayType(new PtrType(new DoubleType()),3), "rows"),
   new GlobalDecl(new DoubleType(), "rows", mkConst(5.5)),
   new GlobalDecl(new ArrayType(new DoubleType(),3), "rows"),
   new GlobalDecl(new ArrayType(new DoubleType(),3), "rows", mkUnaryOp(Minus, mkConst(5.5))))
